src/core/logical_layers/renderers/ClickableTilesRenderer/ClickableTilesRenderer.tsx

```
@@ -0,0 +1,307 @@
+    return mapSource;
+  }
+
+  /* Active and hover feature state */
+  protected _borderLayerId?: string;
+  resetFeatureStates?: () => void;
+  async addHoverAndActiveFeatureState(map: ApplicationMap, style: LayerStyle | null) {
+    await mapLoaded(map);
+    const sourceId = this._sourceId;
+
+    /* Create layer for hover / active effect and add to map */
+    const borderLayerId = sourceId + '_border';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)